changeSdkVersion.ps1

```diff
@@ -26,7 +26,7 @@ if (-Not ($version -match "\d+\.\d+\.\d+")) {
 $packagesJson.version = $version
 $packagesJson = $packagesJson | ConvertTo-Json 
 $packagesJson = $packagesJson -replace "\\u0026", "&"
-$packagesJson | Out-File $packageJsonPath
+[System.IO.File]::WriteAllLines($packageJsonPath, $packagesJson)
```

Comment:
Files are incorrectly encoded and GitHub is showing them as binary. WriteAllLines will save them as UTF-8-BOM.